docs/makeup-hoist-element/index.js

```diff
@@ -0,0 +1,11 @@
+const Hoist = require('../../packages/makeup-hoist-element');
+
+const elementToHoist = document.querySelector('#hoist-this');
+
+elementToHoist.addEventListener('click', function() {
+    if (Hoist.isHoisted(elementToHoist)) {
```

Comment:
I would remove the `isHoisted` method and just let the app/consumer track state of whether they hoisted the element or not.